gtsfm/densify/mvs_patchmatchnet.py

```diff
@@ -176,7 +185,18 @@ def densify(
             min_num_consistent_views=min_num_consistent_views,
         )
 
-        return dense_point_cloud, dense_point_colors
+        # Initialize densify metrics, add elapsed time per batch to the metric list
+        densify_metrics = GtsfmMetricsGroup(
+            name=METRICS_GROUP,
+            metrics=[
+                GtsfmMetric(name="num_valid_reference_views", data=len(loader)),
+                GtsfmMetric(name="elapsed_time_per_ref_img(sec)", data=batch_times),
+            ],
+        )
+        # merge filter matrics to densify metrics
+        densify_metrics.extend(filter_metrics)
```

Comment:
nit: prefer the name `filtering_metrics`. `filter_metrics` sounds like a function name

Reply:
thanks for updating `filter_metrics` to the new name below. can you update here too?

Reply:
sure. I forgot it.